docs/components/input.md

```diff
@@ -260,6 +260,7 @@ export default defineComponent({
 ## Slots
 
 - `icon` - slot for icon on the left or right of the input (position can be controlled through [prop](#iconposition))
+- `label-tooltip` - slot for tooltip content if input has a label and label has tooltip (note: this slot is not available for `overlayLabel`)
```

Comment:
question: should we need to point out that the slot overrides the `label-attributes`?